### PR TITLE
Docs: Include required severity value in array

### PR DESCRIPTION
**Pull request by helloRupa**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ X ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
I added the missing numerical value in the array value for this rule. The rule does not work without it.

#### Is there anything you'd like reviewers to focus on?
No, it's one character.

### CODE REVIEW

File: docs/rules/id-denylist.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For example, to restrict the use of common generic identifiers:
 
 ```json
 {
-    "id-denylist": ["error", "data", "err", "e", "cb", "callback"]
+    "id-denylist": [2, "error", "data", "err", "e", "cb", "callback"]
```
Comment (paulsmithkc) [inline docs/rules/id-denylist.md:30]: Don't these identifiers need to be strings?

Comment (mdjermanovic) [inline docs/rules/id-denylist.md:30]: `"error"` is the severity, `"data"`, `"err"`, `"e"`, `"cb"` and `"callback"` are the options (the names of restricted identifiers)

**Review by mdjermanovic — COMMENTED** [review]
Hi @helloRupa, thanks for the PR!

> I added the missing numerical value in the array value for this rule.

Severity can be a numerical value `0`, `1`, `2`, or a string value `"off"` (same as `0`), `"warn"` (same as `1`), `"error"` (same as `2`).

> The rule does not work without it.

I tried the `"id-denylist": ["error", "data", "err", "e", "cb", "callback"]` configuration and it works - the rule disallows `data`, `err`, `e`, `cb` and `callback` identifiers.

**Comment by nzakas** [comment]
Closing, as the documentation is correct.

**Pull request closed without merging**